src/process-columnar-results.js

```diff
@@ -75,6 +75,12 @@ export default function processColumnarResults(
               )
               break
             case "FLOAT":
+              const value = data.columns[c].data.arr_col[r].data.real_col[e]
```

Comment:
Remind me why we aren't doing this for DOUBLE too?

Reply:
DOUBLE already matches the mapdql output from what I can tell